### PR TITLE
Write logs of the openshift-baremetal-install output

### PR DESCRIPTION
**Pull request by mulbc**
# Description

This makes it easier to watch the current install status and debug.
Logs will be written to the bootstrap server (not VM)

Fixes # (issue)

## Type of change

Please select the appropiate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- I deployed this code 3x using the Alias lab's hardware. It worked flawlessly and allowed me to watch the install progress.

**Test Configuration**:

- Versions: OCP 4.5.2 (stable)
- Hardware: bare-metal Alias lab (Dell servers)

## Checklist

- [ x ] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not being merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

File: ansible-ipi-install/roles/installer/tasks/60_deploy_ocp.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@
   block:
     - name: Run OpenShift Cluster install as async task
       shell: |
-        /usr/local/bin/openshift-baremetal-install --dir {{ dir }} --log-level debug create cluster
+        /usr/local/bin/openshift-baremetal-install --dir {{ dir }} --log-level debug create cluster > {{ dir }}/create-cluster.stdout 2> {{ dir }}/create-cluster.stderr
```
Comment (smalleni) [inline ansible-ipi-install/roles/installer/tasks/60_deploy_ocp.yml:20]: Thanks for your contribution @mulbc but I thought the logs are captured in /home/kni/.openshift_install.log on the provisioner host, by default. How is this different?

Comment (mulbc) [reply ansible-ipi-install/roles/installer/tasks/60_deploy_ocp.yml:20]: That file does not exist for me...

```
[root@e24-h23-740xd ~]# cd /home/kni/
[root@e24-h23-740xd kni]# ls -al
total 28
drwx------  8 kni  kni   220 Jul 20 15:31 .
drwxr-xr-x. 3 root root   17 Jul 20 10:25 ..
drwx------  3 kni  kni    17 Jul 20 10:25 .ansible
drwxrwxr-x  2 kni  kni   254 Jul 20 15:36 .ansible_async
drwxr-xr-x  8 root root  233 Jul 20 12:47 badfish
-rw-------  1 kni  kni   347 Jul 20 11:43 .bash_history
-rw-r--r--  1 kni  kni    18 Aug 30  2019 .bash_logout
-rw-r--r--  1 kni  kni   141 Aug 30  2019 .bash_profile
-rw-r--r--  1 kni  kni   371 Jul 20 10:43 .bashrc
drwx------  4 kni  kni    44 Jul 20 10:43 .cache
drwxr-xr-x  4 kni  kni  4096 Jul 20 16:28 clusterconfigs
-rw-r--r--  1 root root  245 Jul 20 13:10 hosts.txt
drwxr-xr-x  2 kni  kni    80 Jul 20 11:34 .ssh
-rw-r--r--  1 root root   98 Jul 20 13:35 workers.txt
[root@e24-h23-740xd kni]# ls -al /home/kni/.openshift_install.log
ls: cannot access '/home/kni/.openshift_install.log': No such file or directory
```

Comment (smalleni) [reply ansible-ipi-install/roles/installer/tasks/60_deploy_ocp.yml:20]: My bad, it's /home/kni/clusterconfigs/.openshift_install.log

Comment (mulbc) [reply ansible-ipi-install/roles/installer/tasks/60_deploy_ocp.yml:20]: That's a bit too late - I don't have the scale lab environment any more... but I assume that log is written once Ansible is done, not while Ansible is working?
If so, my change will allow the user to watch the openshift-install output during the install to see at what stage the install is, if the IPI is doing some actions currently or to immediately see where errors are.

For me this was very nice and I thought others would enjoy it as well

Comment (smalleni) [reply ansible-ipi-install/roles/installer/tasks/60_deploy_ocp.yml:20]: Sorry about the delay, I was busy in other OpenStack related projects. No, the log is written realtime when ansible is running.

Comment (mulbc) [reply ansible-ipi-install/roles/installer/tasks/60_deploy_ocp.yml:20]: Do you get the output of openshift-install in that log?

Example:
```DEBUG OpenShift Installer 4.5.4                    
DEBUG Built from commit 01f5643a02f154246fab0923f8828aa9ae3b76fb 
DEBUG Fetching Install Config...                   
DEBUG Loading Install Config...                    
DEBUG   Loading SSH Key...                         
DEBUG   Loading Base Domain...                     
DEBUG     Loading Platform...                      
DEBUG   Loading Cluster Name...                    
DEBUG     Loading Base Domain...                   
DEBUG     Loading Platform...                      
DEBUG   Loading Pull Secret...                     
DEBUG   Loading Platform...                        
DEBUG Using Install Config loaded from state file  
DEBUG Reusing previously-fetched Install Config    
INFO Waiting up to 30m0s for the cluster at https://api.bench.chris.ocs.ninja:6443 to initialize... 
DEBUG Still waiting for the cluster to initialize: Cluster operator console is reporting a failure: RouteHealthDegraded: route not yet available, https://console-openshift-console.apps.bench.chris.ocs.ninja/health returns '503 Service Unavailable' 
DEBUG Still waiting for the cluster to initialize: Working towards 4.5.4 
DEBUG Still waiting for the cluster to initialize: Working towards 4.5.4: downloading update 
DEBUG Still waiting for the cluster to initialize: Working towards 4.5.4 
DEBUG Still waiting for the cluster to initialize: Working towards 4.5.4: 2% complete 
DEBUG Still waiting for the cluster to initialize: Working towards 4.5.4: 7% complete 
DEBUG Still waiting for the cluster to initialize: Working towards 4.5.4: 11% complete 
DEBUG Still waiting for the cluster to initialize: Working towards 4.5.4: 12% complete 
DEBUG Still waiting for the cluster to initialize: Working towards 4.5.4: 43% complete 
DEBUG Still waiting for the cluster to initialize: Working towards 4.5.4: 67% complete 
DEBUG Still waiting for the cluster to initialize: Working towards 4.5.4: 72% complete 
DEBUG Still waiting for the cluster to initialize: Working towards 4.5.4: 86% complete 
DEBUG Still waiting for the cluster to initialize: Cluster operator marketplace has not yet reported success 
DEBUG Still waiting for the cluster to initialize: Working towards 4.5.4: 87% complete 
```

if so, I'll close this ticket

Comment (smalleni) [reply ansible-ipi-install/roles/installer/tasks/60_deploy_ocp.yml:20]: Yes,
```time="2020-08-04T15:42:44Z" level=debug msg="Built from commit 01f5643a02f154246fab0923f8828aa9ae3b76fb"
time="2020-08-04T15:42:44Z" level=debug msg="Fetching Install Config..."
time="2020-08-04T15:42:44Z" level=debug msg="Loading Install Config..."
time="2020-08-04T15:42:44Z" level=debug msg="  Loading SSH Key..."
time="2020-08-04T15:42:44Z" level=debug msg="  Loading Base Domain..."
time="2020-08-04T15:42:44Z" level=debug msg="    Loading Platform..."
time="2020-08-04T15:42:44Z" level=debug msg="  Loading Cluster Name..."
time="2020-08-04T15:42:44Z" level=debug msg="    Loading Base Domain..."
time="2020-08-04T15:42:44Z" level=debug msg="    Loading Platform..."
time="2020-08-04T15:42:44Z" level=debug msg="  Loading Pull Secret..."
time="2020-08-04T15:42:44Z" level=debug msg="  Loading Platform..."
time="2020-08-04T15:42:44Z" level=debug msg="Using Install Config loaded from state file"
time="2020-08-04T15:42:44Z" level=debug msg="Reusing previously-fetched Install Config"
time="2020-08-04T15:42:44Z" level=info msg="Waiting up to 1h0m0s for the cluster at https://api.test167.myocp4.com:6443 to initialize..."
time="2020-08-04T15:42:45Z" level=debug msg="Still waiting for the cluster to initialize: Working towards 4.5.4: 69% complete"
time="2020-08-04T15:42:45Z" level=debug
time="2020-08-04T15:42:45Z" level=debug msg="Terraform has been successfully initialized!"
time="2020-08-04T15:42:45Z" level=debug
time="2020-08-04T15:42:45Z" level=debug msg="You may now begin working with Terraform. Try running \"terraform plan\" to see"
time="2020-08-04T15:42:45Z" level=debug msg="any changes that are required for your infrastructure. All Terraform commands"
time="2020-08-04T15:42:45Z" level=debug msg="should now work."
time="2020-08-04T15:42:45Z" level=debug
time="2020-08-04T15:42:45Z" level=debug msg="If you ever set or change modules or backend configuration for Terraform,"
time="2020-08-04T15:42:45Z" level=debug msg="rerun this command to reinitialize your working directory. If you forget, other"
time="2020-08-04T15:42:45Z" level=debug msg="commands will detect it and remind you to do so if necessary."
/Built from

```

Apologies for the wrong location of the file earlier and delay in getting back. Thanks for taking the time to contribute!

**Comment by mulbc** [comment]
Closing since `/home/kni/clusterconfigs/.openshift_install.log` already provides this

**Pull request closed without merging**